sirepo/package_data/static/js/hellweg.js

```diff
@@ -404,10 +404,17 @@ SIREPO.app.directive('summaryTable', function() {
             function parseSummaryRows(summaryText) {
                 var text = summaryText.replace(/^(\n|.)*RESULTS\n+==+/, '').replace(/==+/, '');
                 var label = null;
+                var meanEnergy;
                 $scope.summaryRows = [];
                 text.split(/\n+/).forEach(function(line) {
                     line.split(/\s*=\s*/).forEach(function(v) {
                         if (label) {
+                            if (label == 'Average Energy') {
+                                meanEnergy = Number(v.replace(/MeV/, ''));
+                            }
+                            if (label == 'Energy Spectrum (FWHM)') {
+                                v = energySpectrumFromPercentage(v, meanEnergy);
+                            }
```

Comment:
Is 'Average Energy' guaranteed to come before 'Energy Spectrum'? If not `meanEnergy` may not be defined.

Reply:
Good point. I'll write it to get meanEnergy and set Energy Spectrum after that first loop